lib/models/avatar-cache.js

```diff
@@ -0,0 +1,38 @@
+import {Emitter} from 'event-kit';
+
+const LOADING_AVATAR_URL = 'https://github.com/simurai.png';
```

Comment:
@simurai: 😇 

Think you could replace this with a suitable placeholder that we can show while we're waiting for the real avatar URL to come back from the GitHub API?

Reply:
Maybe it could be the same as the default avatar: https://github.com/atom/github/pull/1322#discussion_r170824537

I'll try to add that.